src/vizier/services/metadata/controllers/k8smeta/k8s_metadata_handler_test.go

```diff
@@ -981,6 +981,7 @@ func TestPodUpdateProcessor_GetUpdatesToSend(t *testing.T) {
 					UID:              "ijkl",
 					Name:             "object_md",
 					Namespace:        "",
+					Labels:           `{"app":"myApp1", "project":"myProj1"}`,
```

Comment:
the test fixture for this test doesn't have labels, so this test fails. can you please fix this test?
you can run `bazel test //src/vizier/services/metadata/controllers/k8smeta:k8smeta_test` to run the test locally
`bazel test //src/vizier/services/metadata/controllers/k8smeta:k8smeta_test --test_output=streamed` is more convenient when actively developing/fixing.

Reply:
Fixed with https://github.com/pixie-io/pixie/pull/468/commits/4c274415c0861836c56b3a4bf08a5da08e612ea0
Thanks for the guidance